esmvaltool/diag_scripts/shared/plot/__init__.py

```diff
@@ -1,16 +1,14 @@
 """Module that provides common plot functions."""
 
-from ._plot import (
-    get_path_to_mpl_style,
-    get_dataset_style,
-    quickplot,
-    multi_dataset_scatterplot,
-    scatterplot,
-)
+from ._plot import (get_dataset_style, get_path_to_mpl_style, global_contourf,
+                    global_pcolormesh, multi_dataset_scatterplot, quickplot,
+                    scatterplot)
```

Comment:
Please keep the trailing comma (or even better, use our new [pre-commit](https://docs.esmvaltool.org/en/latest/community/introduction.html#pre-commit) configuration to have it inserted automatically)

Reply:
I will run `pre-commit` on all files. Note that I already used an autoformatter some time ago, but this kept changing the code over and over (when the version changed), so at some point I stopped using them.

Reply:
Got a similar issue again: `yapf` is formatting something in a way so that `flake8` complains about it:

```
esmvaltool/diag_scripts/shared/io.py:245:9: E125 continuation line with same indent as next logical line
```

This is really not user-friendly....

Reply:
And it removes empty lines at the end of function docstring as given in the [`numpy`](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) example. Sorry, I will not use them and only run `isort`.